.github/actions/ginkgo/main-scheduled.yaml

```diff
@@ -1,7 +1,7 @@
 # This file contain which k8s-versions should be executed for on a regular basis
 ---
 k8s-version:
+  - "1.31"
```

Comment:
We need to drop this change as we want to tests 1.31 for PRs.

Reply:
yes, therefore adding it to this file :smile: